mollie/api/objects/organization.py

```diff
@@ -43,3 +43,7 @@ def vat_regulation(self):
     @property
     def dashboard(self):
         return self._get_link("dashboard")
+
+    # TODO: Implement https://docs.mollie.com/reference/v2/organizations-api/get-partner
```

Comment:
Is this TODO intentional?

Reply:
Yes, but the change is not in scope for this PR (or a blocker for an upcoming release).